pkg/volume/cinder/attacher.go

```diff
@@ -293,10 +329,14 @@ func (detacher *cinderDiskDetacher) Detach(deviceMountPath string, nodeName type
 	}
 
 	if err = detacher.cinderProvider.DetachDisk(instanceID, volumeID); err != nil {
```

Comment:
So if we return timeout error, then the detach controller will retry again later.  Do we still need to check if operation is pending?  Or can the cinder provider handle duplicate detach calls?